go/flags/endtoend/vttablet.txt

```diff
@@ -195,7 +195,6 @@ Usage of vttablet:
       --enable_lag_throttler                                             If true, vttablet will run a throttler service, and will implicitly enable heartbeats
       --enable_query_plan_field_caching                                  (DEPRECATED) This option fetches & caches fields (columns) when storing query plans (default true)
       --enable_replication_reporter                                      Use polling to track replication lag.
-      --enable_semi_sync                                                 DEPRECATED - Set the correct durability policy on the keyspace instead.
```

Comment:
Deprecated flags don't show up in the output?

Reply:
yep! instead, when you use them (which still works), the deprecation message (second arg to `MarkDeprecated`) is printed to the command's stderr `io.Writer` ([c.f.](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.MarkDeprecated))

Reply:
This is perfect!!